executor/distsql.go

```diff
@@ -251,9 +251,25 @@ func (e *IndexReaderExecutor) Close() error {
 	err := e.result.Close()
 	e.result = nil
 	e.ctx.StoreQueryFeedback(e.feedback)
+	e.collectIndexUsage()
 	return err
 }
 
+func (e *IndexReaderExecutor) collectIndexUsage() {
+	if e.ctx.IndexUsageCollectorActivated() {
```

Comment:
Should we skip this function if it is full index scan?

Reply:
ditto